OracleDatabase/RAC/OracleDNSServer/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+# 
+# Since: November, 2018
+# Author: paramdeep.saini@oracle.com
+# Description: Build script for building Oracle Database RAC Docker images.
+# 
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+# 
+# Copyright (c) 2014-2021 Oracle and/or its affiliates. All rights reserved.
+# 
+
+usage() {
+  cat << EOF
+
+Usage: buildDockerImage.sh -v [version] [-i] [-o] [Docker build option]
+Builds a Docker Image for Oracle Database.
+  
+Parameters:
+   -v: version to build
+       Choose one of: $(for i in $(ls -d */); do echo -n "${i%%/}  "; done)
+   -o: passes on Docker build option
```

Comment:
Remove Docker and rephrase to "pass custom build options to the container runtime."